javascript-data-model/order-history.js

```diff
@@ -0,0 +1,50 @@
+var orderHistory = [
+  {
+    title: 'Javascript for impatient programmers',
+    type: 'book',
+    author: 'Dr. Axel Rauschmayer',
+    totalCost: 34.00,
+    itemCost: 31.55,
+    orderPlaced: 'August 4, 2020',
+    orderDelivered: 'August 8, 2020',
+    returnWindowClosed: 'September 7, 2020',
+    orderNumber: '114-3941689-8772232'
+  },
+  {
+    title: 'The Timeless Way of Building',
+    type: 'book',
+    author: 'Christopher Alexander',
+    totalCost: 44.53,
+    itemCost: 41.33,
+    orderPlaced: 'July 19, 2020',
+    orderDelivered: 'July 20, 2020',
+    returnWindowClosed: 'August 19, 2020',
+    orderNumber: '113-9984268-1280257'
+  },
+  {
+    title: 'Gamecuber Controller Adapter',
+    type: 'video game accessory',
+    author: '',
+    totalCost: 17.22,
+    itemCost: 15.98,
+    orderPlaced: 'July 4, 2020',
+    orderDelivered: 'July 7, 2020',
+    returnWindowClosed: 'August 5, 2020',
+    orderNumber: '114-28575557-9059409'
+  },
+  {
+    title: ['The Art of Sql', 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)'],
+    type: ['book', 'video game accessory'],
+    author: [' Stephanie Faroult', ''],
```

Comment:
Aha! Your format here is different than the other order objects, which is not the desired format. Consistency is key! 

It looks like you have multiple items that you are storing individual data about in separate arrays. Rather than subdividing like this, it might be good to have an array of objects describing each items that was bought. That object could have a property for each piece of information about that item.